MBBSEmu/IO/FileUtility.cs

```diff
@@ -14,14 +16,25 @@ namespace MBBSEmu.IO
     /// </summary>
     public class FileUtility : IFileUtility
     {
-        private readonly ILogger _logger;
+        private static readonly Char[] PATH_SEPARATORS = new char[] {'\\', '/'};
```

Comment:
type keyword is 'char', 'Char' is an alias.

also can use array initializer `= {'\\', '/'};`